meson.build

```diff
@@ -1912,7 +1912,6 @@ libsystemd = shared_library(
                         librt],
         link_depends : libsystemd_sym,
         install : true,
-        install_tag: 'libsystemd',
```

Comment:
this was requested here https://github.com/systemd/systemd/pull/23204#issuecomment-1110977393

Reply:
I might be wrong but I think it was requested in the sense that it would probably be nice to have those tags eventually. It was before it was pointed out how kludgy it is in terms of making it work with all meson versions `systemd` claims to support.

Reply:
> this was requested here [#23204 (comment)](https://github.com/systemd/systemd/pull/23204#issuecomment-1110977393)

It did not clarify explicit usages. It just said 'probably ...'.

From https://github.com/systemd/systemd/pull/23204#issuecomment-1110977393,
> In general, I think this is a very nice feature. We should probably figure out what tags we'd want to use. We'd probably want on for `libsystemd`, maybe also one for the standalone variants of tools, and one for tests, man pages, html pages.